### PR TITLE
Latexify gan mixtureofgaussian

### PR DESCRIPTION
**Pull request by dhruvpatel144**
## Description
Latexified figure 26.7

### Figure Number
26.7

### Figures

| Before PR | After PR |
| :-: | :-: |
|before pr 4|after pr4|

### Issue 
#96

### CODE REVIEW

**Comment by dhruvpatel144** [comment]
Please find the text file for latex template below. 
[latex file for fig 26.7.txt](https://github.com/pyprobml-review/pyprobml-review/files/9020298/latex.file.for.fig.26.7.txt)

File: notebooks/book2/26/gan_mixture_of_gaussians.ipynb [hunk 1]
```diff
@@ -0,0 +1,818 @@
+{
```
Comment (karm-patel) [inline notebooks/book2/26/gan_mixture_of_gaussians.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    %pip install -qq git+https://github.com/google/flax.git
</pre><p>Remove this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (karm-patel) [inline notebooks/book2/26/gan_mixture_of_gaussians.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #12.    import optax
</pre><p>Wrap with try...except</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

**Comment by karm-patel** [comment]
After resolving the above comments you can create a PR in the main repo.
Mention that you are not able to reproduce exact results, and at the 20000th iteration, you were getting only one small mode, so you have decreased the iterations

File: notebooks/book2/26/gan_mixture_of_gaussians.ipynb (hunk 1 above)
[the remark this answers is not in the document]
Comment (dhruvpatel144) [reply notebooks/book2/26/gan_mixture_of_gaussians.ipynb:1]: Implemented it and had mentioned the iteration part in the PR. Kindly check

**Pull request merged**